test/SafeProtocolMediator.spec.ts

```diff
@@ -334,6 +336,89 @@ describe("SafeProtocolMediator", async () => {
                 await expect(tx).to.emit(safeProtocolMediator, "ActionsExecuted").withArgs(safeAddress, safeTx.metaHash, 1);
             });
 
+            it("Should process a SafeTransaction and transfer ETH from safe to an EOA guard enabled", async () => {
```

Comment:
the description here is slightly confusing. should it say "with a guard enabled" instead?

Reply:
Since other tests focus on the pre/post-check terminology I'd use it here too